.github/workflows/create-block.yml

```diff
@@ -39,5 +39,5 @@ jobs:
 
             - name: npm install, build, format and lint
               run: |
-                  npm ci
+                  npm install
```

Comment:
Can you explain why is `npm install` necessary with React 0.64?

Reply:
Sorry, I should've mentioned this in the description, will add it there shortly after.

`npm ci` was throwing `cb() never called!` errors randomly both on Linux and macOS. We're still not sure what's the reason. 
I tried various different combinations of node, npm, caching, etc. also debugging manually. Those attempts can be traced in some of [these commits](https://github.com/WordPress/gutenberg/commits/rnmobile/try/upgrade-0-64?after=436975285c0002e08cc33b62662e7549eff1e574+34&author=ceyhun&branch=rnmobile%2Ftry%2Fupgrade-0-64). Only thing that made the `cb() never called!` errors go away so far was using `npm install` instead of `npm ci`. I believe this might be an npm bug and hope it will be fixed with npm@7.

Reply:
@youknowriad and @desrosj, do you know how to make those GitHub actions work by keeping `npm ci` that should be faster?

Reply:
I'm not sure this change is what caused the test to start passing, @desrosj made a change on trunk to improve the situation for these failures already. I think we still have some cases left but less than before.